docs/serverless/edr-manage/allowlist-endpoint-3rd-party-av.mdx

```diff
@@ -0,0 +1,66 @@
+---
+slug: /serverless/security/allowlist-endpoint
+title: Allowlist Elastic Endpoint in third-party antivirus apps
+description: Add Elastic Endpoint as a trusted application in third-party antivirus (AV) software.
+tags: [ 'serverless', 'security', 'overview' ]
+status: in review
+---
+
+<DocBadge template="technical preview" />
+<div id="blocklist"></div>
+
```

Comment:
This page could use a similar disambiguation callout & link to the "Trusted apps" page. (And we'd need to add it to the ESS/AsciiDoc version too.)